types/index.d.ts

```diff
@@ -765,9 +777,9 @@ export interface BaseAttachmentUIComponentProps {
 		Examples include canceling a \/giphy command or shuffling the results.
 		*/
   actionHandler?(
-    name: string,
-    value: string,
-    event: React.BaseSyntheticEvent,
+    name: string | Record<string, any>,
```

Comment:
not a huge deal but these will prob need to be unknown rather than any when we go to TS